### PR TITLE
Polychromic Clothing, new loadout reward

### PR DESCRIPTION
**Pull request by SynnGraffkin**
## About The Pull Request

Ports Polychromic clothing in their current state, made by Ghommie, using [Polychromic element](https://github.com/Citadel-Station-13/Citadel-Station-13/pull/11379) as the primary reference for the current state of what to bring here. 

This adds in a set of clothes that can be earned via Metacoins (or whatever we eventually rename it to) that the colors can be changed on the fly. Jumpsuit slot items are 5k points, the winter coat is 5k points, and the cloak is 10k points (variables can be changed)

A side effect of this, which can probably be atomized into its own PR, is that while moving the Under clothing procs from clothing to _under, I also put in Cit's ability to ctrl click your jumpsuit to immediately max your sensors.

Setting DNM label for now until ~~changing the clothing colors ACTUALLY works and doesn't runtime like hell~~ polycloak and wintercoat work.

Primarily putting the PR up now, since there was the debate on cloaks being an option at all, and maybe someone smarter knows how to fix color runtimes.

## Changelog
:cl:
add: Polychromic clothing added to the Loadout coin shop in player preferences. Now we can really see who has a sense of color coordination.
add: Jumpsuit slot items that have suit sensors can now be ctrl clicked to immediately max sensors.
/:cl:

### CODE REVIEW

**Comment by tiramisuapimancer** [comment]
Mald signed off on the horny items?

**Comment by Maldaris** [comment]
> Mald signed off on the horny items?

wait there was more than pantsu?

**Comment by triplezeta** [comment]
what horny items- wait i see them now

**Comment by SynnGraffkin** [comment]
I DID show you both the pantsu and the bottomless shirt. ( ͡° ͜ʖ ͡°)

**Comment by SynnGraffkin** [comment]
Either way, the more horny options can obviously be commented out or at least left in as spawnable objects instead of a loadout item. It's just fun getting initial reactions.

File: code/datums/action.dm [hunk 1]
```diff
@@ -6,12 +6,14 @@
 /datum/action
 	var/name = "Generic Action"
 	var/desc = null
-	var/obj/target = null
+	var/atom/target = null //Wasp - Cit #11379
```
Comment (Superyodeler) [inline code/datums/action.dm:9]: Did you make sure this won't break anything?

**Review by Superyodeler — CHANGES_REQUESTED** [review]
I could access the polychrome objects' color change menu, but none of them actually changed color.
Also, the button for changing color was just a picture of the object, with no button background to make it easier to click.

**Comment by Superyodeler** [comment]
Just noticed the runtime:

[15:09:50] Runtime in ,: bad arg name 'color'
  usr: Guest-767415254/(Timothy Callison)
  usr.loc: (Construction Area (42, 56, 2))

**Comment by SynnGraffkin** [comment]
```Primarily putting the PR up now, since there was the debate on cloaks being an option at all, and maybe someone smarter knows how to fix color runtimes.```

I haven't touched it in a while, as I was occupied by other things.

**Comment by Superyodeler** [comment]
Whoops, I should have read the PR more closely.

**Comment by SynnGraffkin** [comment]
I trust Ghommie as a genius the atom thing shouldn't be an issue, but solving the color argument will be... annoying.

**Comment by dee-dubya** [comment]
Honestly not too keen on this one, it feels like it'll murder the easy to identify departmental identity of someone at a glance if everyone can just wear a gay pride suit after enough playtime.

On the other hand, people can already swap out their uniforms for silly shit as-is, so it's not that big a deal I suppose.

not gonna say no to it or anything, just not particularly looking forward to it

**Comment by SynnGraffkin** [comment]
Thanks @untrius for finding the problem with color arguments. Still gotta get winter coat and cloak working proper but progress 👏

File: code/datums/action.dm (hunk 1 above)
Comment (SynnGraffkin) [reply code/datums/action.dm:9]: Resolved - I was missing the background for the button entirely.

**Comment by tiramisuapimancer** [comment]
Couple of requests here- Cit recently had a fix PR for things like the winter coat where the hood was allegedly bugged and wouldn't polychrome.

The second thing is that can you please make the background to the polychrome action button not animated. It's so distracting on Cit and I hate it so much

**Comment by SynnGraffkin** [comment]
Well, the winter coat thing didn't fix them, as we found out in testing.

**Comment by SynnGraffkin** [comment]
Is not stale, I'm just lazy.

**Pull request closed without merging**